### ISSUE TITLE
Refactor some cmd.js logic out into its own file for easier tests

### ISSUE THREAD

**Issue by lholmquist** [nodejs/core-validate-commit#71]
While working on issue #69 I started to come across places where it was a little difficult to test if just a particular flag was called.   I think it might help if some of the logic that is in `cmd.js` is broken out into its own file, so it would be easier to test if a particular flag was specified, and not necessarily test what that flag actually does.  That could be done in its own separate test, where it would be easier to mock certain functions.

@richardlau @Trott thoughts?

I can send a PR so we can talk about it also

**Comment by richardlau** [comment]
> I can send a PR so we can talk about it also

Yeah, send a PR -- It'll be much easier to discuss/reason. I have no objections to the idea in principle.
